### PR TITLE
More updates 2021 05 11

### PR DESCRIPTION
**Pull request by jaredbeck**
rails 5.2, ruby 2.7

We're getting there.

They get easier the closer one gets to current.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -18,37 +18,18 @@ gem 'jc-validates_timeliness'
...
-# But that should be fixed soon by:
-# https://github.com/heroku/heroku-buildpack-ruby/pull/11
-# Then again, do I really want serve_static_assets?
-group :production, :stage do
-  gem 'rails_12factor'
-end
-
 # rspec-rails wants to be in the :development group
 # to "expose generators and rake tasks"
 group :test, :development do
```
Comment (neagle) [inline Gemfile:35]: When I checked this out, updated my ruby and bundle versions, and tried to run `rails server`, I got a crash with an error about needing to add the gem "listen". I added it to the `:development` group here, and it fixed the issue. Were you able to run things locally without that?

**Review by neagle — COMMENTED** [review]
Other than the specific question about the `listen` gem, this looks great. I was able to get it running locally and did a relatively quick click-through of a bunch of the major functionality in terms of modifying attendees, viewing reports, adding contacts, and things look good.

Only someone who invested a fair number of hours in beginning to try upgrading this can properly appreciate how fantastic it is to see the version numbers flying up. Thanks, @jaredbeck!

File: Gemfile (hunk 1 above)
Comment (jaredbeck) [reply Gemfile]: Good catch. I have been using the `test` environment, so this didn't come up. Added in 3d31cd4.

**Comment by jaredbeck** [comment]
> Only someone who invested a fair number of hours in beginning to try upgrading this can properly appreciate how fantastic it is to see the version numbers flying up. Thanks, @jaredbeck!

Happy to help! I enjoy this kind of work for the same reason. There's so much we do as programmers where the outcome is unclear. Did we win? Will people use it? So, I think I enjoy work like updates, or performance work, because there's a clear, measurable goal.

**Comment by neagle** [comment]
I love that perspective. I enjoy a lot of the more subjective design work involved in feature implementation, but it's true that there's nothing quite as unambiguously satisfying as a platform upgrade.

**Pull request merged**